src/Apps/Auction/AuctionApp.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Separator, Serif } from "@artsy/palette"
+import { AuctionApp_sale } from "__generated__/AuctionApp_sale.graphql"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { trackPageViewWrapper } from "Apps/Order/Utils/trackPageViewWrapper"
+import { ErrorPage } from "Components/ErrorPage"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql, RelayProp } from "react-relay"
+import {
+  Elements,
+  ReactStripeElements,
+  StripeProvider,
+} from "react-stripe-elements"
+import RegistrationForm from "./Components/RegistrationForm"
+
+declare global {
+  interface Window {
+    Stripe?: (key: string) => stripe.Stripe
+    sd: {
+      STRIPE_PUBLISHABLE_KEY: string
```

Comment:
Any reason `sharify.d.ts` doesn't [cover this](https://github.com/damassi/reaction/blob/8a0da9ad42b6cd5ef9986af68a63a804dc29a26e/typings/sharify.d.ts#L38)?